### ISSUE TITLE
Basic PWA setup

### ISSUE THREAD

**Issue by Sekhmet** [busyorg/busy#1491]
[Progressive Web Apps](https://ionicframework.com/pwa) (PWAs) are meant to provide native-like app experience at friction of a size of regular native-app. To make sure app works well (we should aim for 60FPS) a lot of improvements is required. However, everything we do to improve PWA improves experience on mobile and desktop as well, so I think it's worth to work on this.
At first we should make Busy work as PWA, for this we need:
- Manifest file (we need special logo for this as well - see comment below).
- Generate service worker that caches static assets.
- Extend timeout on the client so it won't timeout on slow connections (I think it should stay the same or even go lower on SSR though - 5 seconds is a lot for TTFB).
- Handle no connection on the client and display meaningful message to user.

**Comment by Sekhmet** [comment]
I created logo similar to https://busy.org/images/logo.png (it needs to be 512px at least and it then is served in different sizes).
It looks like this on mobile:
![image](https://user-images.githubusercontent.com/1968722/35923788-860e0860-0c21-11e8-8953-f26bb95ae363.png)
![image](https://user-images.githubusercontent.com/1968722/35923804-8fa40578-0c21-11e8-85af-a039072ce9e4.png)
![image](https://user-images.githubusercontent.com/1968722/35923808-94dc7052-0c21-11e8-8cc4-766e402a48dd.png)

And like this on desktop:
![image](https://user-images.githubusercontent.com/1968722/35923857-b30d193c-0c21-11e8-99bc-538ea4f2313f.png)
![image](https://user-images.githubusercontent.com/1968722/35923866-b9eb61b4-0c21-11e8-8bd1-bd36f4690c3a.png)

I think it doesn't really match Android (and probably iOS as well). What do you think @pareshUD?
Those are guidelines for Android: https://material.io/guidelines/style/icons.html#

**Comment by jm90m** [comment]
cool looks like esteem is also built on ionic framework...

I was telling @bonustrack earlier, that busy runs really well on both my iphones.. I have an iPhone 7+ and an iPhone 5S, and its sometimes faster than my macbook, I was using both to test PostModal PR when my laptop was getting repaired.

**Comment by Sekhmet** [comment]
I don't think we should build PWA using Ionic. We should have the same app, but it should work as PWA as well. It's just great resource to learn about PWAs.

**Comment by bonustrack** [comment]
@pareshUD ??
